src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -240,4 +242,21 @@ public String getId() throws AbortException {
         }
     }
 
+    @Whitelisted
+    public List<Cause> getCauses() throws AbortException {
```

Comment:
Only useful insofar as the `Cause` implementations themselves have whitelisted getters.